unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Tests/EntityInformationControllerShould.cs

```diff
@@ -2,13 +2,15 @@
 using DCL.Components;
 using DCL.Controllers;
 using DCL.Helpers;
+using DCL.Models;
 using NSubstitute;
 using NUnit.Framework;
 using UnityEngine;
+using UnityEngine.SceneManagement;
 
 namespace Tests.BuildModeHUDControllers
 {
-    public class EntityInformationControllerShould
+    public class EntityInformationControllerShould : IntegrationTestSuite_Legacy
```

Comment:
```suggestion
    public class EntityInformationControllerShould : IntegrationTestSuite
```
I don't see why we need to use the legacy suite here.

Reply:
there is a bug in the NSustitute library where the IDCLEntity are not mocked correctly ( for some internal race conditions). After it is fixed, we should go to IntegrationTestSuite but for now this is a workaround for it, however I have added a comment to reflect it